man/systemd.network.xml

```diff
@@ -2012,6 +2012,14 @@ allow my_server_t localnet_peer_t:peer recv;</programlisting>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>UseIPv6OnlyPreferred=</varname></term>
+        <listitem>
+          <para>When true, the DHCPv4 configuration will be delayed by timer provided by the DHCP server.
```

Comment:
Maybe, better to extend the explanation that we will skip to configure IPv4 network, e.g.
`delayed by the timespan provided by the DHCP server and skip to configure dynamic IPv4 network connectivity if IPv6 connectivity is provided within the timespan.`